test/e2e/rollout_test.go

```diff
@@ -4993,6 +5019,312 @@ var _ = SIGDescribe("Rollout", func() {
 			WaitDeploymentAllPodsReady(workload)
 		})
 	})
+
+	KruiseDescribe("DaemonSet canary rollout", func() {
```

Comment:
plz add corresponding workflow setting in .github/workflows, you may refers to e2e-statefulset-1.19.yaml

Reply:
Added .github/workflows/e2e-daemonset-1.19.yaml